bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -722,7 +722,7 @@ mod tests {
 
             let mut builder = BlockData::new_from_geth_data_with_params(
                 block.clone(),
-                CircuitsParams {
+                ConcreteCP {
```

Comment:
Now that we have the dynamic generation of params, we should replace all unit tests that use default params / custom params, by dynamic calculation of params!